ush/parsing_namelists_MOM6.sh

```diff
@@ -25,24 +25,6 @@ EOF
 #  new_lscale=.true.
 #EOF
 
-if [[ "${DO_OCN_SPPT}" == "YES" ]]; then
-  cat >> input.nml <<EOF
-  OCNSPPT=${OCNSPPT:-1.0}
-  OCNSPPT_LSCALE=${OCNSPPT_LSCALE:-500e3}
-  OCNSPPT_TAU=${OCNSPPT_TAU:-21600}
-  ISEED_OCNSPPT=${ISEED_OCNSPPT:-${ISEED}}
-EOF
-fi
-
-if [[ "${DO_OCN_PERT_EPBL}" == "YES" ]]; then
-  cat >> input.nml <<EOF
-  EPBL=${EPBL:-1.0}
-  EPBL_LSCALE=${EPBL_LSCALE:-500e3}
-  EPBL_TAU=${EPBL_TAU:-21600}
-  ISEED_EPBL=${ISEED_EPBL:-${ISEED}}
-EOF
-fi
-
```

Comment:
I do see that this has been moved to `parsing_namelists_FV3.sh`, but consider the case when we may be running a data atmosphere in which case `parsing_namelists_FV3.sh` will not be called.
Would we then have to move those back from `parsing_namelists_FV3.sh` back to here?

Reply:
These namelist options are under &nam_stochy, which is defined in the parsing_namlists_FV.sh file.

Reply:
Thanks for explaining @NeilBarton-NOAA